src/02_execution/04_executor.md

```diff
@@ -17,11 +17,12 @@ In this section, we'll write our own simple executor capable of running a large
 number of top-level futures to completion concurrently.
 
 For this example, we depend on the `futures` crate for the `ArcWake` trait,
-which provides an easy way to construct a `Waker`.
+which provides an easy way to construct a `Waker`. Edit `Cargo.toml` to add
+a new dependency:
 
 ```toml
 [package]
-name = "xyz"
+name = "timer_future"
```

Comment:
Note that the code in the previous section is called as a separate crate.

https://github.com/rust-lang/async-book/blob/1981378435f6026b2322500385051bd9f6998dd5/examples/02_04_executor/src/lib.rs#L16-L17

Reply:
Sorry, I'm not sure I understand your comment. The examples you linked to seem to diverge a bit from the tutorial?
Does that mean we're missing even more steps? Maybe I misunderstood the guide, but there's nowhere else in the guide up to this point where we're doing any other `cargo new`, and it's the `cargo new timer_future` that fills out that `name` field.
I'm totally OK with closing this PR if you feel it adds more confusion than value, though. As a user when I was following the tutorial, I was slightly confused about where I was supposed to add the dependency and which crate the code sample was referring to.

Reply:
Sorry, my review is no correct.
I misunderstood that TimerFuture was implemented in src/main.rs instead of src/lib.rs in the previous section. Then I saw how it was imported in this section (and how the example code was actually written) and misunderstood it to be an import from another crate.